plugin/v2/settable.go

```diff
@@ -0,0 +1,73 @@
+package v2
+
+import (
+	"fmt"
+	"strings"
+)
+
+type settable struct {
+	name  string
+	field string
```

Comment:
@thaJeztah basically I need a tuple to represent a flag passed as`name.field = value`
maybe @dnephin you have an idea ?